dracut/35ignition-edge/ignition-edge-generator

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+# Originally this was known as 'ignition-generator' found in ignition-dracut.
```

Comment:
let's add a permalink?

https://github.com/coreos/ignition-dracut/blob/4b56b584cdd88373c294b50acfd48d0cdd5f2401/dracut/30ignition/ignition-generator

Reply:
actually, I stole this from fedora-coreos-config - I might just drop this as it's not related afaict (I mean, slightly related, but not sure what this comment adds in value here)